### PR TITLE
:chart_with_upwards_trend: Add KPI on API Rest

### PR DESCRIPTION
**Pull request by GurneyHallack**

### CODE REVIEW

**Comment by Hipska** [comment]
Hi Guy, could you elaborate a little?

**Comment by GurneyHallack** [comment]
Hi Thomas,

When searching to optimize a API query (a complexe OQL query with Server and 6 other classes and returning only the name of the server) which takes 0.9 s for 437 elements, I found that they weren't any KPI available, to identify what it's causing the delay.

After implementing some, I found that the SQL query was taking 0.67 s, because it was getting all the fields of Server (#65 created as a result).

I think it would good to be able to analyse the performance of the API. Thus, I added some KPI but feel free to optimize them.

File: webservices/rest.php [hunk 1]
```diff
@@ -118,9 +118,16 @@ function json_last_error_msg() {
 try
 {
 	utils::UseParamFile();
-
+        
+	$oKPI = new ExecutionKPI();
+	$oKPI->ComputeAndReport('Data model loaded');
```
Comment (piRGoif) [inline webservices/rest.php]: Are there any use for this log ?

Comment (GurneyHallack) [reply webservices/rest.php]: Hi,

I based this commit, from https://github.com/Combodo/iTop/blob/develop/pages/UI.php
Line 354 for "Data model loaded"
Line 353 & 356 for recreating the mode.

I could update my code to use only one instance (so creating before the Try/Catch)

Comment (piRGoif) [reply webservices/rest.php]: Ok ! The lines with ExecutionKPI in UI.php are quite old and not the best way to do :)

You could keep the 'Data model loaded' trace, but we would prefer only one ExecutionKPI instance to be used. Commit this and I'll merge !

Thanks !

**Comment by piRGoif** [comment]
Hello,
Thanks for the modifications, ok to merge !

**Pull request merged**